src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/BuildUpToDateCheck.cs

```diff
@@ -19,11 +19,18 @@ namespace Microsoft.VisualStudio.ProjectSystem
     [ExportMetadata("BeforeDrainCriticalTasks", true)]
     internal sealed class BuildUpToDateCheck : OnceInitializedOnceDisposed, IBuildUpToDateCheckProvider
     {
+        private enum CopyType
```

Comment:
I know, I know, but it's such a *small* enum. Guess it's time to bust out a subdirectory.